components/cluster/pcs/files/pcsd.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0'?>
 <!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
 <service_bundle type='manifest' name='PCSD_WebGUI'>
-  <service name='application/pcs' type='service' version='1'>
+  <service name='application/pcsd' type='service' version='1'>
```

Comment:
Why do we rename existing service? This can break users.

Reply:
I realized that in linux systems the service has the name "pcsd".
And thought it would be better now to make this correction.
But I don't insist on the change.
Rollback to 'pcs'?

Reply:
I'd say, yes.